openpype/tools/workfiles/app.py

```diff
@@ -823,7 +823,11 @@ def on_save_as_pressed(self):
             return
 
         # Trigger before save event
-        BeforeWorkfileSave.emit(work_filename, self._workdir_path)
+        emit_event(
+            "before.workfile.save",
+            {"filename": work_filename, "workdir_path": self._workdir_path},
```

Comment:
That is really good question ;) I would say that it was not used in `after.workfile.save` callbacks and because the change of arguments required change of all registerd callbacks it was not an easy task. Which now is...